dspace-api/src/test/java/org/dspace/content/DuplicateDetectionIT.java

```diff
@@ -56,7 +55,6 @@ public class DuplicateDetectionTest extends AbstractIntegrationTestWithDatabase
 
     private static final Logger log = LogManager.getLogger();
 
-    @Before
```

Comment:
@vins01-4science : Why was this `@Before` annotation removed?  Wouldn't that mean that this `setUp()` method is no longer being called before every test?

Reply:
That's because we're already overriding the method setup, so we don't need to duplicate that annotation.

I've seen this problem with other tests too, but as you can see from the pipeline, the method has been executed correctly.

Reply:
Pushed the changes on this class @tdonohue . Let me know if you find something else.